src/mixins/SearchMixin.js

```diff
@@ -1,26 +0,0 @@
-import store from '@/store';
-
-export default {
-  methods: {
-    // TODO: Make method less hacky and better/less ubiquitous name
```

Comment:
done and done and gotten rid of the mixin, which should be considered an antipattern